corehq/apps/userreports/fixtures.py

```diff
@@ -51,16 +51,22 @@ def __call__(self, user, version, last_sync=None):
 
         root = ElementTree.Element('fixture', attrib={'id': self.id})
         reports_elem = ElementTree.Element('reports')
+        used_report_configs = []
```

Comment:
Took me a moment to realize you were just using this as a counter. This might be more clearly acheived with 

```
report_id_usage_counts = defaultdict(int)
...
    report_config._index = report_id_usage_counts[report_id]
    report_id_usage_counts[report_id] += 1
```